test/UBAFeeCalculator.test.ts

```diff
@@ -1,23 +1,23 @@
-import { expect } from "chai";
-import { bnZero, toBN } from "../utils";
-import UBAFeeConfig from "./UBAFeeConfig";
+import { bnZero, toBN } from "../src/utils";
+import UBAFeeConfig from "../src/UBAFeeCalculator/UBAFeeConfig";
 import {
   computePiecewiseLinearFunction,
   getBounds,
   getDepositBalancingFee,
   getInterval,
   getRefundBalancingFee,
   performLinearIntegration,
-} from "./UBAFeeUtility";
-import { FlowTupleParameters } from "./UBAFeeTypes";
+} from "../src/UBAFeeCalculator/UBAFeeUtility";
+import { FlowTupleParameters } from "../src/UBAFeeCalculator/UBAFeeTypes";
 import { parseUnits } from "ethers/lib/utils";
-import { UBA_BOUNDS_RANGE_MAX, UBA_BOUNDS_RANGE_MIN } from "../constants";
+import { UBA_BOUNDS_RANGE_MAX, UBA_BOUNDS_RANGE_MIN } from "../src/constants";
+import { expect } from "./utils";
 
 describe("UBA Fee Calculations", () => {
   let config: UBAFeeConfig;
   let tuples: FlowTupleParameters;
 
-  beforeAll(() => {
+  beforeEach(() => {
```

Comment:
Could this be a `before` ?

Reply:
https://github.com/across-protocol/sdk-v2/pull/408#discussion_r1341592151